docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -2996,8 +3417,150 @@ Path to the file that contains:
 
 **Example**
 
-``` xml
-<users_config>users.xml</users_config>
+```xml
+<users_config>
+    users.xml
+</users_config>
+```
+
+## validate_tcp_client_information
+
+Determines whether validation of client information is enabled when a query packet is received.
+
+By default, it is `false`:
+
+```xml
+<validate_tcp_client_information>
+    false
+</validate_tcp_client_information>
+```
+
+## users_without_row_policies_can_read_rows
```

Comment:
The settings described in l. 3428-3563 can't be used stand-alone, they need to be sub-tags of tag `<access_control_improvements>`. We should move them into a common section and describe them in a table.